cfme/tests/intelligence/reports/test_crud.py

```diff
@@ -264,3 +263,26 @@ def test_reports_delete_saved_report(appliance, request):
     view.configuration.item_select(
         item='Delete selected Saved Reports', handle_alert=True)
     assert not report.exists
+
+
+def test_reports_create_schedule_for_base_report_once(appliance, request):
```

Comment:
Required:
This should be a crud operation.
Please create and delete this scheduled report properly and assert for both
remove finalizer